include/ibamr/FirstOrderStokesWaveGenerator.h

```diff
@@ -0,0 +1,59 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (c) 2020 - 2020 by the IBAMR developers
+// All rights reserved.
+//
+// This file is part of IBAMR.
+//
+// IBAMR is free software and is distributed under the 3-clause BSD
+// license. The full text of the license can be found in the file
+// COPYRIGHT at the top level directory of IBAMR.
+//
+// ---------------------------------------------------------------------
+
+/////////////////////// INCLUDE GUARD ////////////////////////////////////
+
+#ifndef included_FirstOrderStokesWaveGenerator
+#define included_FirstOrderStokesWaveGenerator
+
+///////////////////////////// INCLUDES ///////////////////////////////////
+
```

Comment:
```suggestion
#include <ibamr/config.h>

```
All of the headers now include this so that our standard macros are always available.